AudioCompression/src/audioCompression/algorithm/FilterBankStep.java

```diff
@@ -82,6 +82,8 @@ public RawAudio reverse(Subbands input) {
 		}
 		WavAudioOutput out = 
 				new WavAudioOutput(windows, input.getWindowOverlap(), input.getSampleRate());
+		
+		out.writeFile(name);
```

Comment:
This isn't a huge deal to me.  One reason I want to include it is because an invocation of the reverse of the pipeline "should" complete the entire process of decompressing the given file and creating the Wav file for use.  Now, that doesn't mean we need to call the write function in the filterbank, but what I would propose is that we move that write function to happen at the end of the constructor for the WavAudioOutput.  That would be similar to the constructor of the WavAudioInput reading in the wav file in its' constructor.  

Otherwise, it means we need to remember to call the write function each time we decompress something.

Either way, we just need to be clear how to get the decompressed wav file.